admin/src/components/ContributionMessages/ContributionMessagesList.vue

```diff
@@ -57,15 +78,36 @@ const props = defineProps({
   },
 })
 
-const emit = defineEmits(['update-status', 'reload-contribution', 'update-contributions'])
+const emit = defineEmits([
+  'update-status',
+  'reload-contribution',
+  'update-contributions',
+  'search-for-email',
+])
```

Comment:
if you define the emit-string here, why do you not use them in the emit-statements below?
for example in line 134 and 138

Reply:
https://vuejs.org/guide/components/events.html